### ISSUE TITLE
backquote will be removed (using `init`)

### ISSUE THREAD

**Issue by soranoba** [nicklockwood/SwiftFormat#682]
## Problems

```swift
enum Flag: String, Codable {
   case `init`
}

let flags: [Flag] = [.`init`]
```
Execute swiftformat to this code will remove the back quotes.
Backquote must not be deleted because init is reserved.

## Environments

```
$ swiftformat --version
0.45.0
```

```
$ cat .swiftformat 
# format options

 # file options
--exclude */modules, Pods, Carthage, fastlane, GeneratedMocks.swift

 # rules
--disable trailingClosures, wrapMultilineStatementBraces
--stripunusedargs closure-only

 # version
--swiftversion 5
```

**Comment by nicklockwood** [comment]
@soranoba thanks for reporting this, I'll fix it in the next release. In the meantime, you can work around it by adding the following comment in the line above:

```swift
// swiftformat:disable:next redundantBackticks
let flags: [Flag] = [.`init`]
```

Or just disable the `redundantBackticks` rule globally in your SwiftFormat configuration.

**Comment by soranoba** [comment]
Thank you for the quick response.

**Comment by nicklockwood** [comment]
@soranoba fixed in 0.45.2.
